project.clj

```diff
@@ -1,43 +1,43 @@
-(defproject re-frame "0.4.1"
-  :description  "A Clojurescript MVC-like Framework For Writing SPAs Using Reagent."
-  :url          "https://github.com/Day8/re-frame.git"
-  :license      {:name "MIT"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/clojurescript "0.0-3211"]
+(defproject re-frame "0.4.2-TRANSDUCERS"
+  :description "A Clojurescript MVC-like Framework For Writing SPAs Using Reagent."
+  :url "https://github.com/Day8/re_frame.git"
```

Comment:
ah, that was unintentional, I was renaming a folder under tests directory and intellij was too smart, I guess. Will change it back.